js/fieldToConceptIdMapping.js

```diff
@@ -35,6 +36,12 @@ export default
     "hipaaVersion": 412000022,
     "userProfileHistory": 569151507,
     "userProfileUpdateTimestamp": 371303487,
+    "profileChangeRequestedBy": 611005658,
+    "firebaseSignInMechanism": 995036844,
```

Comment:
are we using these four constants in the code anywhere? also we don't do either `signInGoogle` or `signInPassword` anymore, I forgot which one is email and which one is username / password

Reply:
`hipaaVersion: 412000022` - this one I'm not sure of.
    `userProfileHistory: 569151507` - this is the newly added constant for history (yes, in use).
    `userProfileUpdateTimestamp: 371303487` - this is the history timestamp (yes, in use).
    `"profileChangeRequestedBy": 611005658` - this was just added on Nicole's request (yes, in use).
    `"firebaseSignInMechanism": 995036844` - according to the conceptId search this resolves to the `signInGoogle`, `signInPhone`, or `signInPassword` cIds.

Maybe the `signIn` cIds are outdated. I noticed Abhinav just used `'phone'` and `'email'` instead of using a conceptId for this in SMDB. Is that the route you want me to take here? So `firebaseSignInMechanism` is in use, but I can avoid the `signIn` cIds. Thanks and please let me know.

Reply:
I'm merging this so it's available, will continue this discussion and update accordingly with the next PR. Thanks.